sources/install/package_forensic.sh

```diff
@@ -53,7 +53,7 @@ function install_volatility3() {
     git -C /opt/tools/ clone --depth 1 https://github.com/volatilityfoundation/volatility3
     pipx install /opt/tools/volatility3
     # volatility's setup.py installs requirements from requirements-minimal.txt. Some reqs from requirements.txt are missing, injecting now
-    pipx inject volatility3 yara-python capstone pycryptodome
+    pipx inject volatility3 yara-python capstone
```

Comment:
We need to keep `pycryptodome` here as they are just the missing libs from `requirements-minimal.txt.`
Maybe it would be better to "inject" `requirements.txt` in the pipx venv :shrug: